climada/util/coordinates.py

```diff
@@ -318,6 +570,56 @@ def dist_approx(lat1, lon1, lat2, lon2, log=False, normalize=True,
         raise KeyError("Unknown distance approximation method: %s" % method)
     return (dist, vtan) if log else dist
 
+def dist_great_circle_allgeoms(gdf):
```

Comment:
This function computes total lengths and perimeters of (collections of) geometrical objects. I wouldn't use `dist` in the name of this function because that would imply computing the distance *between* geometrical objects. As an alternative name, I would suggest `geodesic_lengths` or `compute_geodesic_lengths`.